src/components/home/solid-light-system/index.astro

```diff
@@ -1,4 +1,116 @@
 ---
-
+//IMOPORT ATOMS
+import Heading2 from "../../../atoms/solidlight-systems/heading/two/index.astro";
+import Heading4 from "../../../atoms/solidlight-systems/heading/four/index.astro";
+import Heading5 from "../../../atoms/solidlight-systems/heading/five/index.astro";
+import Heading6 from "../../../atoms/solidlight-systems/heading/six/index.astro";
+import Line from "../../../atoms/solidlight-systems/gradient-line/index.astro";
+import List from "../../../atoms/solidlight-systems/lists/list/index.astro";
```

Comment:
@IamDikshaR This should be called ListItem and not List. Imports are done using "@/path" prefix. You are using "../path". Change all the imports in the components you have created.